bitbots_vision/src/bitbots_vision/vision.py

```diff
@@ -274,11 +282,17 @@ def _dynamic_reconfigure_callback(self, config, level):
             [config['blue_color_detector_upper_values_h'], config['blue_color_detector_upper_values_s'],
              config['blue_color_detector_upper_values_v']])
 
-        self.field_color_detector = color.PixelListColorDetector(
-            self.debug_printer,
-            self.package_path,
-            config,
-            primary_detector=True)
+        if config['dynamic_color_space_active']:
```

Comment:
Redundant deactivation of the dynamic color space using this config parameter.

Reply:
closed with 39a9cd0